specifications/api/swagger.yaml

```
@@ -387,6 +387,26 @@ x-ScriptValue: &ScriptValue
               items:
                 $ref: "#/components/schemas/ScriptValue"
 
+    - title: Active from
+      type: object
+      required:
+        - active_from
+      properties:
+        active_from:
+          description: Transaction has to occur no sooner than a specified slot number.
+          type: integer
+          minimum: 0
+
+    - title: Active until
+      type: object
+      required:
+        - active_until
+      properties:
+        active_until:
+          description: Transaction has to occur before a specified slot number.
+          type: integer
+          minimum: 0
```

In the ledger code, these are called `RequireTimeStart` and `RequireTimeExpire`. Any reason to use a different term?

```suggestion
    - title: Active from slot
      type: object
      required:
        - active_from
      properties:
        active_from:
          description: Transaction is only valid starting at the specified slot number (`≥ active_from`).
          type: integer
          minimum: 0

    - title: Active until slot
      type: object
      required:
        - active_until
      properties:
        active_until:
          description: Transaction is only valid before the specified slot number (`< active_until`).
          type: integer
          minimum: 0
```

---

some time ago we thought active_from and active _until are somewhat better, easier ....

---

OK yes they are better names.